README.md

```diff
@@ -1,2 +1,19 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+a) The algorithm looks for the same numbers from the list, but it must not be the same position.
+if it finds similar numbers, it returns true. Otherwise, returns that it didn't find anything.
+
+b) Because it checks the same numbers twice.
+
+c & d) It still runs correct, but it won't go over the same checks every time the first loop "i" increases.
+
+e) Yes. As it checks twice as less as the first example, the time complexity equals to 0,5n^2, which is still quadrataic.
+
+f) https://wiki.python.org/moin/TimeComplexity (First list). Time complexity of Python's *sort* is O(n log n)
+
+g) The time complexity of this algorithm is O(n), because it checks with the numbers that are next to each other, unlike the first one.
```

Comment:
However the algorithm executes a sort, so by part (f) the overall time complexity is O(n log n + n). The first term dominates, so the complexity is O(n log n)